tests/acceptance/learning/test_fault_tolerance.py

```diff
@@ -71,97 +65,3 @@ def bad_bytestring_of_known_nodes():
     warning = warnings[1]['log_format']
     assert str(blockchain_teacher) in warning
     assert "Failed to verify MetadataResponse from Teacher" in warning  # TODO: Cleanup logging templates
-
-
-@pytest.mark.skip("See Issue #1075")  # TODO: Issue #1075
-def test_invalid_operators_tolerance(testerchain,
```

Comment:
Another instance of uncertainty here - even though this test is skipped this seems like an important characteristic to test.  What's the rationale for removing it?

Reply:
I agree, thanks for advancing it.